pinot-controller/src/test/java/org/apache/pinot/controller/helix/core/sharding/SegmentAssignmentStrategyTest.java

```diff
@@ -255,15 +253,20 @@ public void testRealtimeReplicaGroupPartitionAssignment() {
 
     Map<String, String> streamConfigMap = new HashMap<>();
     String type = "kafka";
+    String consumerFactoryClass = "com.test.TestConsumerFactoryClass";
```

Comment:
looks like this class does not exist.

Reply:
It doesn't need to exist unless the test is starting PartitionConsumers. It's just serving as a dummy value. We've been using the kafka class names, just because we had access to them

Reply:
Fixed